utils/docker/images/install-pmdk.sh

```diff
@@ -38,9 +38,9 @@ set -e
 
 git clone https://github.com/pmem/pmdk
 cd pmdk
-git checkout 9b8a2bec65c9eb4922841d53d696463a503842ee
+git checkout stable-1.5
```

Comment:
"stable-1.5" is a moving target, please use the latest commit id from this branch

Reply:
Done.